package/network/utils/xdp-tools/patches/001-configure-remove-Werror.patch

```diff
@@ -0,0 +1,20 @@
+--- a/configure
++++ b/configure
+@@ -196,7 +196,7 @@ int main(int argc, char **argv) {
+     return 0;
+ }
+ EOF
+-    compile_cmd="$CC -o $TMPDIR/libbpftest $TMPDIR/libbpftest.c -Werror $LIBBPF_CFLAGS $LIBBPF_LDLIBS"
++    compile_cmd="$CC -o $TMPDIR/libbpftest $TMPDIR/libbpftest.c $LIBBPF_CFLAGS $LIBBPF_LDLIBS"
```

Comment:
What is this fixing? That `-Werror` is there for a reason: we want this check to fail if the compiler spits out a warning so we can properly feature-detect libbpf features...

Reply:
`configure` doesn't pick the system libbpf if this `-Werror` is in place, see:
```
[...]
config.Lk4udD/libbpftest.c: In function 'main':
config.Lk4udD/libbpftest.c:12:5: error: 'bpf_set_link_xdp_fd_opts' is deprecated: libbpf v0.8+: use bpf_xdp_attach() instead [-Werror=deprecated-declarations]
   12 |     (void) bpf_set_link_xdp_fd_opts(0, 0, 0, &xlopts);
      |     ^
In file included from config.Lk4udD/libbpftest.c:1:
/usr/src/lede/staging_dir/target-aarch64_cortex-a53_musl/usr/include/bpf/libbpf.h:872:16: note: declared here
  872 | LIBBPF_API int bpf_set_link_xdp_fd_opts(int ifindex, int fd, __u32 flags,
      |                ^~~~~~~~~~~~~~~~~~~~~~~~
cc1: all warnings being treated as errors
```

Reply:
This should be fixed by https://github.com/xdp-project/xdp-tools/pull/200

Reply:
Yes, now the upstream source builds as-is.

Reply:
Yes, that fixed it.